diff --git a/server/container_create_linux.go b/server/container_create_linux.go
index 5e3614d..dc1fa92 100644
--- a/server/container_create_linux.go
+++ b/server/container_create_linux.go
@@ -91,7 +91,6 @@ func makeAccessible(path string, uid, gid int, doChown bool) error {
 // makeMountsAccessible makes sure all the mounts are accessible from the user namespace
 func makeMountsAccessible(uid, gid int, mounts []rspec.Mount) error {
        for _, m := range mounts {
-               logrus.Warnln("debug mount:", m)
                if m.Type == "bind" || util.StringInSlice("bind", m.Options) {
                        if err := makeAccessible(m.Source, uid, gid, false); err != nil {
                                return err
