diff --git a/server/container_create_linux.go b/server/container_create_linux.go
index dc1fa9206..9ff4cb52b 100644
--- a/server/container_create_linux.go
+++ b/server/container_create_linux.go
@@ -36,6 +36,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
+	"golang.org/x/sys/unix"
 
 	"github.com/intel/goresctrl/pkg/blockio"
 )
@@ -69,6 +70,20 @@ func makeAccessible(path string, uid, gid int, doChown bool) error {
 		}
 	}
 	for ; path != "/"; path = filepath.Dir(path) {
+
+		var fsinfo unix.Statfs_t
+		unix.Statfs(path, &fsinfo)
+
+		/*
+			Skip CHMOD for NFS mounts
+			On https://man7.org/linux/man-pages/man2/statfs.2.html
+			"NFS_SUPER_MAGIC" = 0x6969
+			0x6969 which converts in 26985
+		*/
+		if fsinfo.Type == 26985 {
+			continue
+		}
+
 		st, err := os.Stat(path)
 		if err != nil {
 			if os.IsNotExist(err) {
